tox.ini

```diff
@@ -35,3 +33,6 @@ minversion=2.0
 addopts= -rxsX 
 pep8ignore = E501 E128 E127
```

Comment:
are the errors listed here still ignored with the new invocation? 
In principle a large refactoring/code-shifting PR should try to avoid also changing semantics ...